### PR TITLE
Feature/mdr 290

### PR DESCRIPTION
**Pull request by mclaurentiu**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -187,6 +187,44 @@ services:
     networks:
       - mydefault
 
+  rdf-fingerprinter-api:
+    container_name: rdf-fingerprinter-api
+    image: meaningfy/rdf-fingerprinter-api:latest
+    ports:
+      - ${RDF_FINGERPRINTER_API_PORT}:${RDF_FINGERPRINTER_API_PORT}
+    env_file: .env
+    restart: always
```
Comment (cosletmihai) [inline docker/docker-compose.yml:196]: this indicated that if the container "breaks" and quits execution, the `docker-compose` will automatically try to restart it?

**Review by cosletmihai — APPROVED** [review]

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -187,6 +187,44 @@ services:
...
+    image: meaningfy/fuseki:latest
+    env_file:
+      - .env
+    environment:
+      - ADMIN_PASSWORD=${RDF_FINGERPRINTER_FUSEKI_ADMIN_PASSWORD}
+      - JVM_ARGS=${RDF_FINGERPRINTER_FUSEKI_JVM_ARGS}
+    ports:
+      - ${RDF_FINGERPRINTER_FUSEKI_PORT}:3030
+    volumes:
+      - ${PWD}/${RDF_FINGERPRINTER_FUSEKI_DATA_FOLDER}/fuseki:/fuseki
```
Comment (costezki) [inline docker/docker-compose.yml:223]: just realised that all the Fuseki data folders must be moved into dedicated volumes defined in volumes section

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -187,6 +187,44 @@ services:
...
+    env_file: .env
+    restart: always
+    networks:
+      - mydefault
+
+  rdf-fingerprinter-ui:
+    container_name: rdf-fingerprinter-ui
+    image: meaningfy/rdf-fingerprinter-ui:latest
+    ports:
+      - ${RDF_FINGERPRINTER_UI_PORT}:${RDF_FINGERPRINTER_UI_PORT}
```
Comment (costezki) [inline docker/docker-compose.yml:204]: I will be surprised if this works. Fuseki, internally, always runs on 3030.

File: tests/config.py [hunk 4]
```diff
@@ -5,5 +5,6 @@
 
 import os
 
-VALIDATOR_UI_URL = os.environ.get('FILENAME', 'http://localhost:8010')
-DIFFER_UI_URL = os.environ.get('FILENAME', 'http://localhost:8030')
+VALIDATOR_UI_URL = os.environ.get('VALIDATOR_UI_URL', 'http://localhost:8010')
```
Comment (costezki) [inline tests/config.py:8]: can we please rename this ENV variable into RDF_VALIDATOR ... , both here in the config.py and in the .env file

**Review by costezki — APPROVED** [review]
well done, only that the compose file and the .env files might need minor updates

File: docker/docker-compose.yml (hunk 1 above)
Comment (costezki) [reply docker/docker-compose.yml:196]: or if the host system is restarted, then docker containers shall shart on boot

**Review by costezki — APPROVED** [review]

**Pull request merged**